modules/imaging_uploader/jsx/UploadForm.js

```diff
@@ -223,19 +255,88 @@ class UploadForm extends React.Component {
       (this.state.uploadProgress > -1) ? "btn btn-primary hide" : undefined
     );
 
-    const notes = "File name must be of type .tgz or tar.gz or .zip. " +
-      "Uploads cannot exceed " + this.props.maxUploadSize;
+    const notes = (
+        <span>
+          File cannot exceed {this.props.maxUploadSize}<br/>
+          File must be of type .tgz or tar.gz or .zip<br/>
+          For files that are not Phantom Scans, file name must begin with
+          <b> [PSCID]_[CandID]_[Visit Label]</b><br/>
+          For example, for CandID <i>100000</i>, PSCID <i>ABC123</i>, and
+          Visit Label <i>V1</i> the file name should be prefixed by:
+          <b> ABC123_100000_V1</b><br/>
+        </span>
+    );
+
+    // Returns individual form elements
+    // For CandID, PSCID, and Visit Label, disabled and required
+    // are updated depending on Phantom Scan value
+    // For all elements, hasError and errorMessage
+    // are updated depending on what values are submitted
     return (
       <div className="row">
         <div className="col-md-7">
           <h3>Upload an imaging scan</h3>
           <br/>
           <FormElement
             name="upload_form"
-            formElements={form}
             fileUpload={true}
-            onUserInput={this.onFormChange}
+            ref="form"
```

Comment:
can't see where this `ref` is necessary & string refs are deprecated. if unused, remove. otherwise use the updated [callback pattern](https://github.com/aces/Loris/blob/minor/modules/datadict/jsx/dataDictIndex.js#L81)